en/PlanView/PlanView.md

```diff
@@ -1,38 +1,33 @@
 # Plan View
 
-![](../../images/plan/PlanView.jpg)
+![](../../images/plan/PlanView.png)
 
-The Plan View is used to plan autonomous missions for you Vehicle. Once the mission is planned and sent to the vehicle, you switch to the [Fly View](../FlyView/FlyView.md) to fly the mission.
+The *Plan View* is used to plan autonomous missions for your Vehicle. Once the mission is planned and sent to the vehicle, you switch to the [*Fly View*](../FlyView/FlyView.md) to fly the mission.
```

Comment:
FYI @barzanisar I generally only use italic to mark up views IFF they aren't already marked up in some other way. So not really needed in this case.